src/hotspot/share/runtime/os.hpp

```diff
@@ -343,7 +345,7 @@ class os: AllStatic {
   static void   commit_memory_or_exit(char* addr, size_t size,
                                       size_t alignment_hint,
                                       bool executable, const char* mesg);
-  static bool   uncommit_memory(char* addr, size_t bytes);
+  static bool   uncommit_memory(char* addr, size_t bytes, bool exec);
```

Comment:
Please make this default=false too, and revert the hunks where this is explicitly set to false